Jenkinsfile

```diff
@@ -35,30 +37,8 @@ pipeline {
             echo 'DRY RUN: Skipping release tag creation'
           }
           else {
-            // get a list of all files changes since the last tag
-            files = sh(returnStdout: true, script: "git diff --name-only HEAD `git describe --match \"v*\" --abbrev=0`").trim()
-            echo "Files changed:\n${files}"
-
-            // the release build process changes the version file, so it will always be changed
-            // if there are other files changed, increment the version and create a new tag
-            if (files != 'version.sbt') {
-              echo 'Running sbt-release'
-
-              // sbt release doesn't like running without these
-              sh(returnStdout: true, script: "git config remote.origin.fetch +refs/heads/*:refs/remotes/origin/*")
-              sh(returnStdout: true, script: "git config branch.main.remote origin")
-              sh(returnStdout: true, script: "git config branch.main.merge refs/heads/main")
-
-              echo sh(returnStdout: true, script: "echo y | sbt \"release with-defaults\"")
```

Comment:
We no longer use `sbt release` since we don't use service's internal version for releases.